docs/business/Setup_Payment.md

```diff
@@ -65,8 +64,58 @@ Once you are set, you can connect your Bazar account to your Stripe account.
 ![Select processor](../assets/Business/Set%20up%20payment-4-create.png) 
 
 
+# Connect to your Paypal Account
+
+[Paypal](https://paypal.com/) is a very widely used payment tool that is very accessible in many countries. It can be good to offer it additionally to your Stripe payment method to avoid conflicts with Payments.
+
+## Requirements: Paypal Business Account
+To use Paypal for the Bazar, you need an active Paypal Business Account. 
+If you don’t have one, [create one](https://www.paypal.com/bizsignup/#/checkAccount) and make sure to verify with your email address.
+
+## Step 1: Add a payment method
+
+- Login to the [seller dashboard](https://newbazar.preciousplastic.com/my_admin_panel.php)
+- Go to Administration/ Payment methods.
+- Click on ‘+’- to add one.
+
+![Payment methods](../assets/Business/Set%20up%20payment-1-payment%20methods.png)
+
+
+## Step 2: Select processor (Paypal)
+
+- Select "Paypal Express Checkout" as processor
+- Then click on ‘configure’
+- 
+![Payment methods](../assets/Business/Set%20up%20paypal%20(1).png)
+
+## Step 3: Fill in Technical Details (1)
+- Scroll down to the **Technical Details** field, this is where you’ll have to insert information from your Paypal Account
+- Find your **Merchant ID** in your Paypal Account under [Business Information](https://www.paypal.com/businessmanage/account/aboutBusiness)
+- Copy it over into your Technical Details
+
+![Payment methods](../assets/Business/Set%20up%20paypal%20(2).png)
+
+## Step 4: Fill in Technical Details (2)
+- Request your **Username, Password and Signature** in [Manage API Access](https://www.paypal.com/businessprofile/mytools/apiaccess/firstparty)
+- Copy it over into your Technical Details
+**When copying over names and numbers, make sure you're only copying the numbers & letters, and NOT empty spaces.**
+
+![Payment methods](../assets/Business/Set%20up%20paypal%20(3).png)
 
-## You're now open for business 🎉
+## Step 5: Fill in Technical Details (3)
+- Select “Live” mode 
+- Check the box to show the Paypal option in the Checkout
+
+![Payment methods](../assets/Business/Set%20up%20paypal%20(5).png)
+
+## Step 6: Finish!
+- Click  ‘Create’
+- you should now see your payment method listed
+
+![Payment methods](../assets/Business/Set%20up%20paypal%20(4).png)
```

Comment:
Would be better to not have spaces in filename. Results into weird code. Using - or _ would be preffered

Reply:
yes, noted, will do in the future!
Should this be live now? I don't see any changes on the page https://community.preciousplastic.com/academy/business/Setup_Payment